styles2.css

```diff
@@ -0,0 +1,547 @@
+li,
+figure,
+figcaption,
+blockquote,
+dl,
+dd,
+input {
+    margin: 0;
+}
+
+/* Выставляем основные настройки по-умолчанию для body */
+body {
+    min-height: 100vh;
+    scroll-behavior: smooth;
+    text-rendering: optimizeSpeed;
+    line-height: 1.5;
+}
+
+/* Удаляем стандартную стилизацию для всех ul и il, у которых есть атрибут class*/
+ul[class],
+ol[class] {
+    list-style: none;
+}
+
+/* Элементы a, у которых нет класса, сбрасываем до дефолтных стилей */
+a:not([class]) {
+    text-decoration-skip-ink: auto;
+}
+
+/* Упрощаем работу с изображениями */
+img {
+    max-width: 100%;
+    display: block;
+}
+
+/* Указываем понятную периодичность в потоке данных у article*/
+article>*+* {
+    margin-top: 1em;
+}
+
+/* Наследуем шрифты для инпутов и кнопок */
+input,
+button,
+textarea,
+select {
+    font: inherit;
+}
+
+/* Удаляем все анимации и переходы для людей, которые предпочитай их не использовать */
+@media (prefers-reduced-motion: reduce) {
+    * {
+        animation-duration: 0.01ms !important;
+        animation-iteration-count: 1 !important;
+        transition-duration: 0.01ms !important;
+        scroll-behavior: auto !important;
+    }
+}
+
+.desktop_menu {
+    display: none;
+}
+
+.top_photo {
+    padding-top: 36px;
+}
+
+.header {
+    display: flex;
+    justify-content: space-between;
+    box-shadow: 0px 1px 0px rgba(0, 0, 0, 0.16);
+}
+
+.top_padding_nordic {
+    padding: 24px 0px 21px 20px;
+}
+
+.top_padding_burger {
+    padding: 24px 24px;
+
+}
+
+.top_section {
+    text-align: center;
+    padding-top: 52px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    position: relative;
+}
+
+.about_blog {
+    font-family: 'New York Extra Large', sans-serif;
+    font-size: 34px;
+    font-weight: 700;
+    line-height: 120%;
+    padding: 0px 20px 0px 20px;
+    margin-right: auto;
+    margin-left: auto;
+    max-width: 375px;
+}
+
+
+
+
+.small_text_about_blog {
+    padding: 24px 20px 0px 20px;
+    font-family: 'SF Mono';
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 170%;
+    text-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+    margin-right: auto;
+    margin-left: auto;
+    max-width: 375px;
+}
+
+.second_section {
+    display: block;
+    margin-top: 24px;
+    padding-left: 20px;
+    padding-right: 20px;
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 640px;
+}
+
+.author_of_post_info {
+    display: flex;
+    padding-top: 24px;
+    border-top: 2px solid #000000;
+    flex-wrap: wrap;
+}
+
+.author_name {
+    font-family: 'SF Pro Display', sans-serif;
+    font-weight: 700;
+    font-size: 13px;
+    line-height: 16px;
+    letter-spacing: 0.05em;
+    text-transform: uppercase;
+}
+
+.author_date {
+    font-family: 'SF Pro Display', sans-serif;
+    font-weight: 400;
+    font-size: 13px;
+    line-height: 16px;
+}
+
+
+.text_about_author {
+    padding-left: 16px;
+}
+
+.table_with_photo {
+    padding-top: 24px;
+    display: block;
```

Comment:
again `block` is the default value for block elements. do research what is the default `display` value for different html elements